### PR TITLE
add(library): Vanilla-Components

### PR DESCRIPTION
**Pull request by ijpatricio**
| Name                 | Link                 |
| -------------------- | -------------------- |
| Vanilla Components | [Resource link](https://vanilla-components.com)|

Vanilla Components
Build Better & Faster
A lightweight, flexible & customizable UI library for Vue 3, styled with Tailwind CSS.

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,6 +254,7 @@
 
 - 💙🧩 [Tailwind UI](https://tailwindui.com) - Component library made with Tailwind CSS.
 - 💙📚 [Headless UI](https://github.com/tailwindlabs/headlessui) - Completely unstyled, fully accessible UI components.
+- 📚 [Vanilla Components](https://vanilla-components.com) - A lightweight, flexible & customizable UI library for Vue 3, styled with Tailwind CSS.
```
Comment (innocenzi) [inline README.md:257]: Should be below "Headless UI float". Also please adapt the description to match the guidelines.

**Review by innocenzi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ijpatricio) [reply README.md]: Thanks. I just submitted the requested changes.

**Comment by ijpatricio** [comment]
Let me know what you think @innocenzi

File: README.md [hunk 2]
```diff
@@ -265,6 +264,7 @@
 - 📚 [Svelte Headless UI](https://github.com/rgossiaux/svelte-headlessui) - Unofficial Svelte port of Headless UI.
 - 📚 [Xtend UI](https://xtendui.com/) - Tailwind CSS components with advanced interactions and animations.
 - 📚 [Headless UI Float](https://headlessui-float.vercel.app) - Floating UI integration for Headless UI.
+- 📚 [Vanilla Components](https://vanilla-components.com) - A great set of Vue components styled with Tailwind CSS and customizable via `tailwind.config.js` using configuration and utility classes.
```
Comment (innocenzi) [inline README.md:267]:
```suggestion
- 📚 [Vanilla Components](https://vanilla-components.com) - Set of fully customizable Vue components.
```

**Comment by innocenzi** [comment]
Hey, I rephrased the description: it wasn't respecting guidelines and I removed the "Tailwind CSS" parts, since this is a Tailwind CSS list it should be obvious. Thanks for the awesome component library!

**Pull request merged**